docs/README.md

```diff
@@ -346,6 +346,7 @@ The following npm scripts are supported in this directory:
 | `build:api-documentation` | Convert package API reports (`.api.json` files) into Markdown. |
 | `build:md-magic` | Updates generated content in Markdown files. |
 | `build:md-magic:code` | `node markdown-magic-code.js` |
+| `build:redirects` | `node ./build-redirects.js` |
```

Comment:
```suggestion
| `build:redirects` | Creates a data file that the redirection azure function has access to. |
```

Don't use that text as is but any text you add here will be preserved by the markdown generator, so you can add documentation to the table.